next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationsHeader.tsx

```diff
@@ -1,9 +1,11 @@
 import cx from 'classnames'
-import { useGlobalStateContext } from 'components/forms/states/GlobalState'
+import { ApplicationsListVariant } from 'components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationsSection'
```

Comment:
this was probably autoimport not doing a great job (we may need to tweak the setting), but it would be nicer to have the path local:

```suggestion
import { ApplicationsListVariant } from './MyApplicationsSection'
```